scripts/recreate-tag.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+# This script recreates tag v1 in the repository
+# It will:
+# 1. Fetch all tags and prune them
+# 2. Checkout the main branch
+# 3. Delete the tag v1
+# 4. Push the deletion of the tag v1
+# 5. Create the tag v1
+
+
+TAG_NAME="v1"
+
+git fetch --tags --prune-tags -f
```

Comment:
I usually force push from local but should be the same
```
git checkout origin/main
git tag -d $TAG_NAME
git tag v1
git push origin tag v1 -f
```

Reply:
This is meant to be run from local (and it's forced `-f`), so yes, I guess it's the same 😅